doc/user/content/release-notes.md

```diff
@@ -57,45 +57,133 @@ Wrap your release notes at the 80 character mark.
+  collision](/sql/identifiers#keyword-collision) documentation for details.
 
   **Backwards-incompatible change.**
 
-- Add `upper` and `lower` to the [string function](/sql/functions#string-func) suite.
+- Change the return type of [`sum`](/sql/functions/#aggregate-func) over
+  [`bigint`](/sql/types/integer)s from `bigint` to
+  [`numeric`](/sql/types/numeric). This avoids the possibility of overflow when
+  summing many large numbers {{% gh 5218 %}}.
 
-- Change [`sum`](/sql/functions/#aggregate-func) over `bigint` to return `numeric`.
-  Previously, `sum` over `bigint` returned `bigint`.
+  We expect the breakage from this change to be minimal, as the semantics
+  of `bigint` and `numeric` are nearly identical.
 
   **Backwards-incompatible change**
 
+- Speed up parsing of [`real`](/sql/types/float) and
+  [`numeric`](/sql/types/numeric) values by approximately 2x and 100x,
+  respectively {{% gh 5341 5343 %}}.
+
+- Add the [`SHOW OBJECTS`](/sql/show-objects) SQL statement to display all
+  objects in a database, regardless of their type.
+
+- Ensure the first batch of updates in a [source](/sql/create-source) without
+  consistency information is stamped with the current wall clock time, rather
+  than timestamp `1` {{% gh 5201 %}}.
+
+- When Materialize consumes a message from a [Kafka source](/sql/create-source/avro-kafka),
+  commit that message's offset back to Kafka {{% gh 5324 %}}. This allows
+  Kafka-related tools to monitor Materialize's consumer lag.
+
+- Improve the PostgreSQL compatibility of several date and time-related
+  functions:
+
+  - Correct `date_trunc`'s rounding behavior when truncating by
+    decade, century, or millenium {{% gh 5056 %}}.
+
+    Thanks to external contributor [@zRedShift](https://github.com/zRedShift).
+
+  - Allow specifying units of `microseconds`, `milliseconds`, `month`,
+    `quarter`, `decade`, `century`, or `millenium` when applying the `EXTRACT`
+    function to an [`interval`](/sql/types/interval) {{% gh 5107 %}}. Previously
+    these units were only supported with the [`timestamp`](/sql/types/timestamp)
+    and [`timestamptz`](/sql/types/timestamptz) types.
+
+    Thanks again to external contributor
+    [@zRedShift](https://github.com/zRedShift).
+
+  - Support multiplying and dividing [`interval`](/sql/types/interval)s by
+    numbers {{% gh 5107 %}}.
+
+    Thanks once more to external contributor
+    [@zRedShift](https://github.com/zRedShift).
+
+  - Handle parsing [`timestamp`](/sql/types/timestamp) and [`timestamptz`](/sql/types/timestamptz)
+    from additional compact formats like `700203` {{% gh 4889 %}}.
+
+- Permit specifying `ALL` as a row count to [`FETCH`](/sql/fetch) to indicate
+  that there is no limit on the number of rows you wish to fetch.
+
+- Add the `upper` and `lower` [string functions](/sql/functions#string-func),
+  which convert any alphabetic characters in a string to uppercase and
+  lowercase, respectively.
```

Comment:
let's group this one with the other new SQL functions (ie, under the "Add the `SHOW OBJECTS` SQL statement) bullet

Reply:
Reordered to keep the SQL stuff closer together!